pyproject.toml

```diff
@@ -44,14 +44,61 @@ archs = ["x86_64", "arm64"]
 [tool.cibuildwheel.linux]
 archs = ["auto", "aarch64"]
 
+[tool.black]
+line-length = 88
+force-exclude = '''
+^/(
+  (
+    # Directories to always exclude (.gitignore is always excluded)
+      \.pyinstaller
+    | docs
+    | examples
+    | astropy/_dev
+    | astropy/_erfa
+    | astropy/config
+    | astropy/constants
+    | astropy/convolution
+    | astropy/coordinates
+    | astropy/cosmology
+    | astropy/extern
+    | astropy/io/ascii
+    | astropy/io/fits
+    | astropy/io/misc
+    | astropy/io/registry
+    | astropy/io/tests
+    | astropy/io/votable
+    | astropy/modeling
+    | astropy/nddata
+    | astropy/samp
+    | astropy/stats
+    | astropy/table
+    | astropy/tests
+    | astropy/time
+    | astropy/timeseries
+    | astropy/uncertainty
+    | astropy/units
+    | astropy/utils
+    | astropy/visualization
+    | astropy/wcs
+  )/
+  # Individual files to always exclude
+  | astropy/io/__init__.py
+  | astropy/__init__.py
+  | astropy/conftest.py
+  | astropy/logger.py
+  | astropy/version.py
+  | conftest.py
+  | setup.py
+)
+'''
+
 [tool.isort]
+    profile = "black"
     extend_skip_glob = [
         "astropy/extern/*",
     ]
-    line_length = 100
     known_third_party = ["erfa", "PyYAML", "packaging", "pytest", "scipy", "matplotlib"]
     known_first_party = ["astropy", "extension_helpers"]
-    multi_line_output = 4
     group_by_package = true
     indented_import_headings = false
     use_parentheses = true
```

Comment:
`use_parentheses` can be removed.
Why do we need `indented_import_headings` ?

Reply:
@nstarman do you know why?

Reply:
Removing `indented_import_headings` setting seemed to have no effect when I ran `isort` so I will remove it.

Reply:
> do you know why?

no, I forget! If it has no effect, then we're good :)